<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InventoryWidget</class>
 <widget class="QWidget" name="InventoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLabel" name="L_INVGOLD">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Bank gold :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="L_BANKGOLD">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Inventory gold :</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QTabWidget" name="TW_INVENTORY">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="T_EQUIPMENT">
      <attribute name="title">
       <string>Equipment</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="ItemTableWidget" name="TW_EQUIPMENT">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::NoDragDrop</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="rowCount">
          <number>20</number>
         </property>
         <property name="columnCount">
          <number>6</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>96</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>96</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>64</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>64</number>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="T_MAIN">
      <attribute name="title">
       <string>Main</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="ItemTableWidget" name="TW_MAIN">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
         <property name="rowCount">
          <number>20</number>
         </property>
         <property name="columnCount">
          <number>6</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>96</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>96</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>64</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>64</number>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="T_ETC">
      <attribute name="title">
       <string>Etc</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="ItemTableWidget" name="TW_ETC">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
         <property name="rowCount">
          <number>20</number>
         </property>
         <property name="columnCount">
          <number>6</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>96</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>96</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>64</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>64</number>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="T_SP">
      <attribute name="title">
       <string>SP</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="ItemTableWidget" name="TW_SP">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
         <property name="rowCount">
          <number>9</number>
         </property>
         <property name="columnCount">
          <number>5</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>96</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>96</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>64</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>64</number>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="T_COSTUMES">
      <attribute name="title">
       <string>Costumes</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="ItemTableWidget" name="TW_COST">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
         <property name="rowCount">
          <number>12</number>
         </property>
         <property name="columnCount">
          <number>5</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>96</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>96</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>64</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>64</number>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="T_MINILAND">
      <attribute name="title">
       <string>Miniland</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="ItemTableWidget" name="TW_MINILAND">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>192</width>
           <height>192</height>
          </size>
         </property>
         <property name="rowCount">
          <number>8</number>
         </property>
         <property name="columnCount">
          <number>7</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>96</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>96</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>64</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>64</number>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemTableWidget</class>
   <extends>QTableWidget</extends>
   <header>modules/itemtablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
